apps/drec-api/migrations/1626300062964-DeviceUpdates.ts

```diff
@@ -0,0 +1,23 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class DeviceUpdates1626300062964 implements MigrationInterface {
+  name = 'DeviceUpdates1626300062964';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "device" ADD "yield" integer NOT NULL DEFAULT '1000'`,
+    );
+    await queryRunner.query(`ALTER TABLE "device" DROP COLUMN "capacity"`);
+    await queryRunner.query(
+      `ALTER TABLE "device" ADD "capacity" integer NOT NULL`,
```

Comment:
so you are dropping capacity column, and recreating it? You sure it's OK?

Reply:
Oh I see, you are changing type. I guess you may consider just going with changing column type sql query, but whatever, it's typescript isn't it? This query however is not impossible to run on table that has some data, because it will fail on NOT NULL (because there will be no data)

Reply:
So, I had to add two more properties and as long as it is not deployed and data is not seeded, I just did a big migration with everything. Should be better now!